nansat/exporter.py

```diff
@@ -124,8 +124,16 @@ def export(self, filename='', bands=None, rm_metadata=None, add_geolocation=True
 
         self.logger.debug('Export - OK!')
 
-    def export2thredds(self, filename, bands={}, metadata=None, mask_name=None, no_mask_value=64, rm_metadata=None,
-                       time=None, created=None):
+    def export2thredds(self,
```

Comment:
please refactor some variable names inside the function to satisfy pep8. For example `srcBands`

Reply:
Done